src/marqo/core/structured_vespa_index/structured_vespa_index.py

```diff
@@ -62,6 +63,13 @@ def to_vespa_document(self, marqo_document: Dict[str, Any]) -> Dict[str, Any]:
 
             index_field = self._marqo_index.field_map[marqo_field]
 
+            if isinstance(marqo_value, (int, float)) and not index_field.type == FieldType.Bool:
```

Comment:
Shouldn't need not index_field.type == FieldType.Bool because boolean conversion to int is done further down in line 73